src/openms/source/KERNEL/MSChromatogram.cpp

```diff
@@ -415,3 +415,52 @@ void MSChromatogram::clear(bool clear_meta_data)
     integer_data_arrays_.clear();
   }
 }
+
+
+OpenMS::MSChromatogram::Iterator set_sum_similar_union(OpenMS::MSChromatogram::Iterator first1,
```

Comment:
should be `setSumSimilarUnion`

Reply:
should this helper be private?

Reply:
its a free function in a cpp... private by design :)